packages/integrations/solid/src/index.ts

```diff
@@ -44,6 +44,8 @@ function getViteConfiguration(isDev: boolean) {
 		},
 		ssr: {
 			external: ['babel-preset-solid'],
+			target: 'node',
+			noExternal: true
```

Comment:
I believe this will make everything in your project be treated as noExternal

Reply:
hi @matthewp yeah you are right, i was too focused on `worker` builds. For a `nodejs` build that does not work as expected. So I rather go with explicit solid-js include (what should be fine for `nodejs` or `worker` ssr build).